dependent_type_theory.md

```diff
@@ -343,15 +358,17 @@ You can also pass types as parameters:
 ```
 The last expression, for example, denotes the function that takes
 three types, ``α``, ``β``, and ``γ``, and two functions, ``g : β → γ``
-and ``f : α → β``, and returns the composition of ``g`` and ``f``.
-(Making sense of the type of this function requires an understanding
-of dependent products, which will be explained below.)
+and ``f : α → β``, and returns the composition of ``g`` and ``f``. For
+example, you could call this function passing the following arguments
+`Nat Nat double Nat double 2` where `double` is defined as `def double
+(x : Nat) : Nat := x + x` (This function is making use of an important
+feature named "Dependent Products", which will be explained below.)
```

Comment:
Let's change "Dependent Products" to "Dependent Types".
The "Dependent Products" here is leftover from previous versions.

Reply:
Ok, sure, so I assume I would also change 903 then that says "Dependent products are also called *sigma* types" to instead say "Dependent types are also called  *sigma* types" ?

Reply:
No, that is not correct. Dependent types are a superset of dependent products/sigma types.

Reply:
I'd suggest to drop these parentheses. The "important feature" has already been described before: "You can also pass types as parameters". This has been connected to DTT before that: "One way in which Lean's dependent type theory extends simple type theory is that types themselves --- entities like Nat and Bool --- are first-class citizens, which is to say that they themselves are objects."

Reply:
Hmm, @Kha  so was the previous statement misleading then: "(Making sense of the type of this function requires an understanding of dependent products, which will be explained below.)".  Perhaps I should just delete the whole sentence?

Reply:
I deleted the sentence.